packages/mobile/src/web3/saga.ts

```diff
@@ -75,6 +81,13 @@ export function* checkWeb3SyncProgress() {
 
         const latestBlock: Block = yield call(getLatestBlock)
         if (latestBlock && latestBlock.number > 0) {
+          ;[meanMillisecs, trackCount] = trackMeanMillisecs(
```

Comment:
If I'm reading this correctly, this is the only event function we need to use trackMeanMilliseconds for, right?

Reply:
okey, CustomEventNames.sync_complete then.

Reply:
okey, perfect -> CustomEventNames.sync_complete